esp-wifi/MIGRATING-0.9.md

```diff
@@ -2,7 +2,7 @@
 
 ## Initialization
 
-You no longer have to set up clocks and pass them to `esp_wifi::initialize`.
+You no longer have to set up clocks and pass them to `esp_wifi::init`.
```

Comment:
The rename needs to be documented separately, and the diffs need to be updated to highlight the rename.